src/main/webapp/app/exam/overview/events/overlay/exam-live-events-overlay.component.ts

```diff
@@ -33,7 +33,7 @@ export class ExamLiveEventsOverlayComponent implements OnInit, OnDestroy {
     eventsToDisplay?: ExamLiveEvent[];
     events: ExamLiveEvent[] = [];
 
-    @Input() examStartDate: dayjs.Dayjs;
+    examStartDate = model<dayjs.Dayjs>();
```

Comment:
Why do we need a `model` here but not in `src/main/webapp/app/exam/overview/events/button/exam-live-events-button.component.ts` that seems to contain a quite similar logic to this component? 

Also shouldn't the model be required?
```suggestion
    examStartDate = model.required<dayjs.Dayjs>();
```

Reply:
We need a model here, because we need to set its value from another component, in this case from `exam-live-events-button.component.ts`:

```
this.alertService.closeAll();
        this.modalRef = this.modalService.open(ExamLiveEventsOverlayComponent, {
            size: 'lg',
            backdrop: 'static',
            animation: false,
            centered: true,
            windowClass: 'live-events-modal-window',
        });

this.modalRef.componentInstance.examStartDate.set(this.examStartDate());
```

But you're right, the modal can be required. I changed that.